gnosis/tabula
# Chore: Infura Environment Variables Naming Update

### PR Description

#### Issue:
closes #260 

During the creation of a publication, users have the option to upload an image before saving. However, the deterministic avatar was generated and displayed instead of the uploaded image (both gif & png formats).

#### Steps to Reproduce:
1. Navigate to publication creation.
2. Upload an image.
3. Create publication.

#### Expected Behavior:
The uploaded image should be displayed upon publication creation.

#### Changes Made:
- Renamed environment variables for clarity and better understanding for developers:
  - `REACT_APP_INFURA_KEY` to `REACT_APP_INFURA_NETWORK_ACCESS_KEY`
  - `REACT_APP_INFURA_API_KEY` to `REACT_APP_INFURA_IPFS_API_KEY`
  - `REACT_APP_INFURA_API_KEY_SECRET` to `REACT_APP_INFURA_IPFS_API_SECRET`

- Updated the `workflows/dev-cd.yml` and `workflows/prod-release-deploy` files to use the new environment variable names.

- Modified `env.example` to reflect the changes in environment variable names.

- In `connectors.ts`, updated the code to use `REACT_APP_INFURA_NETWORK_ACCESS_KEY` and adjusted the error message accordingly.

- In `useIpfs`, made changes to use the newly named environment variables `REACT_APP_INFURA_IPFS_API_KEY` and `REACT_APP_INFURA_IPFS_API_SECRET` and updated the error handling to check for these variables.

#### Files Changed:
- `workflows/dev-cd.yml`
- `workflows/prod-release-deploy`
- `env.example`
- `connectors.ts`
- `useIpfs`

### How to Test:
1. Navigate to publication creation.
2. Upload an image (gif or png).
3. Create a publication and observe the image displayed.

### Notes:
Please ensure to update your local `.env` files with the new environment variable names and their respective values.
